lessons/intro_and_strings/strings.py

```diff
@@ -10,7 +10,7 @@
 # are always preceded by the '#' symbol.
 # Comments don't "run" as part of code.
 # So, for example, I have the following line of code
-# print("Will this line print?")
+print("Will this line print?")
```

Comment:
Notice how git thinks that a small edit is a deletion of a line and addition of a new line.